Lib/test/test_pathlib.py

```diff
@@ -619,6 +619,21 @@ def test_relative_to_common(self):
         self.assertRaises(ValueError, p.relative_to, '')
         self.assertRaises(ValueError, p.relative_to, P('a'))
 
+    def test_is_relative_to(self):
```

Comment:
I would like to see more comprehensive testing. You can imply reuse the various cases from the `relative_to` tests.

Reply:
add all test_relative_to and test_relative_to_common test cases:)